lib/omnibus/logger.rb

```diff
@@ -21,7 +21,7 @@ class Logger
     #
     # @return [Fixnum]
     #
-    LEFT = 40
+    LEFT = 20
```

Comment:
Maybe we should make this value `30`. As you can see from this test build output the alignment is off for a number of lines:
https://gist.github.com/schisamo/bc0ea2673e02d112cc885682eb7a914b